model/Cargo.toml

```diff
@@ -4,7 +4,7 @@ version = "0.1.0"
 edition = "2018"
 
 [dependencies]
-chrono = { version = "0.4", default-features = false, features = ["serde"] }
+chrono = { version = "0.4", default-features = false, features = ["serde", "clock"] }
```

Comment:
why is clock needed?

Reply:
This is used here in `model/order.rs` at line 46.

http://github.com/gnosis/gp-v2-services/blob/456859c8a1c0a76f27f986e5296e8602a4fdff63/model/src/order.rs#L38-L54

The only reason we were getting away with it before is because we already had it when compiling the project as a whole but not when building models alone: ` cargo build -p model`